### DISCUSSION TITLE
Getting startup failure on Actions when deploying the website

### DISCUSSION THREAD

**Discussion by shiranD** [alshedivat/al-folio#2081] [Q&A]
**Acknowledge the following**

- [y] I carefully read and followed the [Getting Started](../../README.md#getting-started) guide.
- [y] I read through [FAQ](../../FAQ.md) and searched through the [past issues](https://github.com/alshedivat/al-folio/issues), none of which addressed my issue.
- [y] The issue I am raising is a potential bug in al-folio and not just a usage question. <br> [For usage questions, please post in the [Discussions](https://github.com/alshedivat/al-folio/discussions) instead of raising an issue.]

**Describe the bug**
This question may be more related to Github Pages than this repo in particular, but perhaps you may be able to help me.

I was able to deploy the website, but at some point, the updates I was providing to master branch stopped being reflected in gh-pages branch ([my website](https://github.com/shiranD/shirand.github.io)). One such update was deleting files*  I don't use in my website such as _pages/repositories.md (or _pages/teaching.md). Ever since I did that, it showed a "startup failure" [here](https://github.com/shiranD/shirand.github.io/actions) for all other actions (except for one). One common comment I see

"[Error: .github#L1](https://github.com/shiranD/shirand.github.io/commit/e82826d65820327f035eb9cf339da63cb643baba#annotation_16983117073)
actions/checkout@v3, docker/setup-qemu-action@v2, docker/setup-buildx-action@v2, docker/login-action@v2, and docker/build-push-action@v4 are not allowed to be used in shiranD/shirand.github.io. Actions in this workflow must be: within a repository owned by shiranD."

I am not sure what the meaning of the comment, and will appreciate any help on this. (I also mistakingly deleted the specific workflow in Actions showing the deletion attempt of _pages/repositories.md, but other ones failed the same).

*which was done through the GitHub api website, and not my local machine

**Comment by george-gca** [comment]
I am trying to imagine if, for some unknown reason, your actions lost the ability to write to your GitHub pages. Try to check again the permission for the GitHub pages. Basically [step 4](https://github.com/alshedivat/al-folio/blob/master/INSTALL.md#installing-and-deploying).

**Comment by shiranD** [comment]
I checked it now, and it seems similar to what it was before, the source is set to "deploy from a branch" and the branch is gh-pages. Thanks!

**Reply by george-gca** [reply]
Can you pinpoint when exactly this started to happen? If not it will be really difficult to help you solve it, then maybe it is best to start fresh and just apply the changes you made on top of a fresh install.
